app/controllers/api/v1/comments_controller.rb

```diff
@@ -50,6 +50,29 @@ def delete
     end
   end
 
+  require "net/http"
+  require "json"
+
+  # NOTE: I have used a temporary LibreTranslate API setup hosted on
```

Comment:
The comment about the temporary LibreTranslate API is clear. However, if this is meant to be a long-term feature, a ticket should be created to replace it with a permanent solution.

Reply:
Ok, I'll raise a ticket for that